modules/sara_r4/cellular_r4.h

```diff
@@ -60,6 +60,9 @@ CellularError_t rebootCellularModem( CellularContext_t * pContext,
                                      bool disablePsm,
                                      bool disableEidrx );
 
+CellularError_t _Cellular_isSockOptSupport( CellularSocketOptionLevel_t optionLevel,
```

Comment:
This function can be a private function since it is only used in SARA_R4 cellular interface implementation.

Reply:
Moved to cellular_r4_api.c and change it to static function.